src/blockchain/ain/jellyfish/raw-tx-util.ts

```diff
@@ -0,0 +1,125 @@
+import { Network } from '@defichain/jellyfish-network';
+import {
+  DeFiTransactionConstants,
+  TransactionSegWit,
+  Witness,
+  WitnessScript,
+  Script,
+  OP_CODES,
+  toOPCodes,
+  Vin,
+  Vout,
+} from '@defichain/jellyfish-transaction';
+import { fromAddress } from '@defichain/jellyfish-address';
+import { Prevout } from '@defichain/jellyfish-transaction-builder';
+import BigNumber from 'bignumber.js';
+import { SmartBuffer } from 'smart-buffer';
+import { AddressUnspent } from '@defichain/whale-api-client/dist/api/address';
+import { Config } from 'src/config/config';
+
+interface OpPushData {
+  type: string;
+  hex: string;
+}
+
+export class RawTxUtil {
+  // --- PARSING --- //
+
+  static parseOwnerAddress(owner: string, network: Network): [Script, string] {
+    const decodedAddress = fromAddress(owner, network.name);
+    const pushData: OpPushData = decodedAddress?.script.stack[1] as any;
+    if (!decodedAddress.script || !pushData.hex) throw new Error('Could not parse owner address');
+    return [decodedAddress.script, pushData.hex];
+  }
+
+  static parseOperatorPubKeyHash(operator: string, network: Network): [Script, string] {
+    const decodedAddress = fromAddress(operator, network.name);
+    const pushData: OpPushData = decodedAddress?.script.stack[2] as any;
+    if (!decodedAddress.script || !pushData.hex) throw new Error('Could not parse operator address');
+    return [decodedAddress?.script, pushData?.hex];
+  }
```

Comment:
These two look very similar 🤔  and they don't seem to be related to owner/operator roles.